dcs/src/main/java/org/trafodion/dcs/server/ServerManager.java

```diff
@@ -83,6 +83,10 @@
     private int maxRestartAttempts;
     private int retryIntervalMillis;
     private String nid = null;
+    private static String mxosrvrKeepaliveStatus;
```

Comment:
can u change mxosrvr to userProg just like the other properties already defined

Reply:
OK.